operations/dd-toc-guide.md

```diff
@@ -185,9 +236,9 @@ Evaluation summary is composed of two parts: the Criteria and the Adoption. The
 
 ## TOC Internal Review
 
-Once the TOC member has completed the Due Diligence, the TOC member tags the TOC on the PR for an TOC internal review. The TOC member should move the project's card on the [Appliction to Move Levels board](https://github.com/orgs/cncf/projects/27/views/9) to "TOC Review".
+Once the TOC member has completed the Due Diligence, they should create a PR in their personal TOC repo and share the link with the TOC for review. The TOC member should move the project's card on the [Appliction to Move Levels board](https://github.com/orgs/cncf/projects/27/views/9) to "TOC Review".
```

Comment:
`create a PR in their personal TOC repo` should be `create a branch in their personal fork of the TOC repo with DD markdown file`.

(Once there is consensus, the branch is used to create a PR in the main TOC repo)